docs/user/aws/install.md

```diff
@@ -0,0 +1,60 @@
+# OpenShift Installation
+
+At this point, you are now ready to perform the OpenShift installation outlined at try.openshift.com and begin at 
```

Comment:
Nit: "now ready" -> "ready".

And replace "at try.openshift.com" with:

```markdown
[here][cloud-install]
...
[cloud-install]: https://cloud.openshift.com/clusters/install
```

Also, I'd rather avoid linking out to those docs from this repo.  Can we link to the root README's quick-start instead?  Or grow local docs for anything that quick-start is missing?

Reply:
Fixed the link part. I'll let the more organic linking come later as the docs are adopted/evolve/formalized. We may want a bit more there for obtaining pull secrets and the like. The cloud install page helps fill in a couple more blanks.